daemon/events.go

```diff
@@ -80,6 +80,29 @@ func (daemon *Daemon) LogNetworkEventWithAttributes(nw libnetwork.Network, actio
 	daemon.EventsService.Log(action, events.NetworkEventType, actor)
 }
 
+// LogDaemonEventWithAttributes generates an event related to the daemon itself with specific given attributes.
+func (daemon *Daemon) LogDaemonEventWithAttributes(action string, attributes map[string]string) {
+	if daemon.EventsService != nil {
+		for _, label := range daemon.configStore.Labels {
```

Comment:
what I meant is that `attributes` already contains a json version of `daemon.configStore.Labels`, this splits the labels and add them on top of it, duplicating the data.